examples/00-systemcoupling/turek_hron_fsi2.py

```diff
@@ -0,0 +1,465 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+""".. _ref_turek_hron_fsi2_example:
+
+Turek-Hron FSI2 Benchmark Example
+---------------------------------
+
+This example is a version of the *Turek-Hron FSI2* case that is
+often used as a benchmark case for System Coupling. This two-way, fluid-structure
+interaction (FSI) case is based on co-simulation of a transient oscillating
+beam with surface data transfers.
+
+- Ansys Mechanical APDL (MAPDL) is used to perform a transient structural analysis.
+- Ansys Fluent is used to perform a transient fluid-flow analysis.
+- System Coupling coordinates the coupled solution involving the above products to
+  solve the multiphysics problem via co-simulation.
+
+**Problem description**
+
+An elastic beam structure is attached to a rigid cylinder. The system
+resides within a fluid filled channel:
+
+.. image:: /_static/turek_hron_case.png
+   :width: 400pt
+   :align: center
+
+The flow is laminar with a Reynolds number of :math:`Re = 100`. The inlet velocity
+has a parabolic profile with a maximum value of :math:`1.5 \cdot \\bar{U}`, where :math:'\\bar{U}'
```

Comment:
@abhishekchitwar in the last formula, this should be back-ticks, not single quotes. Otherwise, the formula doesn't render correctly:

![image](https://github.com/user-attachments/assets/c8312e30-54cd-4eba-a0d7-2b79d8c311f4)

Reply:
@ochernuk, Updated the formula with back tick to render it as expected!